scripts/entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set +ex
+
+ecosystem_list=("openshift" "knative" "kubevirt")
+
+# Run the data collector for given eco systems and no of days to colelct data
+python3 app/run_data_collector.py -eco-systems ${ecosystem_list[*]} -days ${DAYS};
```

Comment:
How to differentiate the ecosystem specific events if you combine all into one? This information is needed when generating a report.

Reply:
As discussed with you code is written such a way we cant differentiate for now, will be taking some story to work on it in future sprint.